cmd/plural/crypto.go

```diff
@@ -51,6 +51,8 @@ const Gitignore = `/**/.terraform
 .vscode
 `
 
+// IMPORTANT
+// Don't wrap Action methods
```

Comment:
If we want to put the full explanation, it's basically:

repo cryptography relies on git smudge and clean filters, which pipe a file into stdin and respond with a new version of the file from stdout.  If we write anything besides the crypto text, it will no longer be decryptable naturally.